store/sqlite/guest_links.go

```diff
@@ -127,17 +129,62 @@ func (s Store) DeleteGuestLink(id picoshare.GuestLinkID) error {
 	return tx.Commit()
 }
 
+func (s Store) DisableGuestLink(id picoshare.GuestLinkID) error {
+	log.Printf("Disable guest link %s", id)
+
+	tx, err := s.ctx.BeginTx(context.Background(), nil)
```

Comment:
Transactions are useful when we have multiple SQL statements, and we want to rollback previous statements if any in the transaction fail.

When there's only one statement, we can just execute it immediately, like in `InsertGuestLink`.

(ditto below)